src/System.Text.Json/src/System/Text/Json/JsonException.cs

```diff
@@ -48,6 +48,14 @@ public JsonException(string message, string path, long? lineNumber, long? bytePo
             Path = path;
         }
 
+        /// <summary>
+        /// Creates a new exception object with serialized data.
+        /// </summary>
+        /// <param name="info">The <see cref="SerializationInfo"/> that holds the serialized object data about the exception being thrown.</param>
```

Comment:
The full DocId should be used:

```suggestion
        /// <param name="info">The <see cref="T:System.Runtime.Serialization.SerializationInfo"/> that holds the serialized object data about the exception being thrown.</param>
```